server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/SonarQubeMonitor.java

```diff
@@ -27,10 +27,13 @@
 import org.sonar.process.ProcessProperties;
 import org.sonar.server.user.SecurityRealmFactory;
 
+import java.io.File;
 import java.util.LinkedHashMap;
 
 public class SonarQubeMonitor extends BaseMonitorMBean implements SonarQubeMonitorMBean {
 
+  static final String BRANDING_FILE_PATH = "web/WEB-INF/classes/com/sonarsource/branding";
```

Comment:
Just to understand, this works because of the overlay of sonarsource branding in sonar-server, right?